app/models/constants/involved_civilian.rb

```diff
@@ -33,9 +33,11 @@ module InvolvedCivilian
 
     FIREARM_TYPES = ['Handgun', 'Rifle', 'Shotgun', 'Other firearm'].freeze
 
-    CUSTODY_STATUS_TYPES = ['Cited and released', 'In custody', 'Fled', 'Deceased', 'None of these'].freeze
+    CUSTODY_STATUS_TYPES = ['Cited and released', 'In custody (other)',
+                            'In custody (W&I section 5150)', 'Fled',
```

Comment:
I think it would look a little better if `'In custody (W&I section 5150)'` goes before `'In custody (other)'`, so that it's immediately clear what "other" refers to.